CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Cromwell Change Log
 
+## 26
+
+* Added a configuration knob under `system.io` to throttle the number of I/O queries that Cromwell makes.
```

Comment:
let's call it an option or setting or something like that and not a knob in public discourse